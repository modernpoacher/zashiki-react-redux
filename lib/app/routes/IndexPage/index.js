import React from 'react' // eslint-disable-line no-unused-vars
import {
  connect
} from 'react-redux'

import {
  RouteMaster,
  Navigation
} from '../../components'

class IndexPage extends RouteMaster {
  render () {
    console.log('(IndexPage)render()')

    return (
      <section>
        <header>
          <h1>Index Page</h1>
        </header>
        <Navigation />
      </section>
    )
  }
}

export default connect(
  (state) => ({
    routeMaster: state.routeMaster
  })
)(IndexPage)
