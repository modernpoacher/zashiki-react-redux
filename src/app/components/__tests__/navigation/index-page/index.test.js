import React from 'react'
import Enzyme, { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Adapter from 'enzyme-adapter-react-16'

import IndexPage from '@modernpoacher/zashiki-react-redux/app/components/navigation/index-page'

Enzyme.configure({ adapter: new Adapter() })

jest.mock('react-router-dom')

describe('@modernpoacher/zashiki-react-redux/app/components/navigation/index-page', () => {
  it('renders', () => {
    expect(toJson(shallow(<IndexPage />)))
      .toMatchSnapshot()
  })
})
