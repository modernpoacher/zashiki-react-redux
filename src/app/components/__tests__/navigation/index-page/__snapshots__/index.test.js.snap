// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@modernpoacher/zashiki-react-redux/app/components/navigation/index-page renders 1`] = `ShallowWrapper {}`;
